godot-bindings/src/lib.rs

```diff
@@ -179,6 +179,18 @@ pub fn emit_godot_version_cfg() {
     }
 }
 
+/// Emit `#[cfg(wasm_nothreads)]` flag when compiling to Wasm with the "experimental-wasm-nothreads" feature.
+pub fn emit_wasm_nothreads_cfg() {
+    println!(r#"cargo:rustc-check-cfg=cfg(wasm_nothreads, values(none()))"#);
+
+    #[cfg(feature = "experimental-wasm-nothreads")]
+    if std::env::var("CARGO_CFG_TARGET_FAMILY")
+        .is_ok_and(|families| families.split(',').any(|family| family == "wasm"))
```

Comment:
I assume you cannot check directly against `wasm` here?

Reply:
I see, maybe add that as a comment.

Also, isn't that env var always set? If yes, you should `.expect()` it rather than `.is_ok_and()`.